doc/release_notes.rst

```diff
@@ -13,6 +13,8 @@ Upcoming Release
 
 **New Features and major Changes**
 
+* Enable DC carrier in the network model and include converters into the model `PR #392 <https://github.com/pypsa-meets-africa/pypsa-africa/pull/392>`__
```

Comment:
This line should go at the end; most likely there are merge conflicts, please check